releasenotes/notes/add-reset-simplification-pass-82377d80dd0081fd.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Added a new transpiler pass, :class:`~ResetAfterMeasureSimplification`,
+    which is used to replace a :class:`~.Reset` operation after a
+    :class:`~.Measure` with a conditional :class:`~.XGate`. For IBM backends
+    this :class:`~.Reset` operation is performed by doing a measurement and
+    then a conditional X gate. So if used with an IBM backend this pass will
+    remove a duplicate implicit :class:`~.Measure` from the :class:`~.Reset`
+    operation.
```

Comment:
We'll want to soften this so it doesn't say all IBM backends, at the very least, if not scrub the mention of our backends.

Reply:
Yeah, I'll loosen the wording here a bit

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8330/commits/0ea73125be520ce9fa33cb8149bc5c2db3daa861